composer.json

```diff
@@ -13,9 +13,9 @@
         }
     ],
     "require": {
-        "php"           : "^7.1 || ^8.0",
+        "php"           : "^7.2 || ^8.0",
```

Comment:
Is there a reason to drop PHP 7,1?
(personally I don't care - but are there installations "out there" on PHP 7.1?)

Reply:
There is no version of Twig that supports both PHP 7.1 and PHP 8.1 - their newer releases dropped 7.1 support. So this change was necessary